deployment/common/changeset/transfer_link_token.go

```diff
@@ -0,0 +1,184 @@
+package changeset
+
+import (
+	"errors"
+	"fmt"
+	"math/big"
+	"time"
+
+	"github.com/ethereum/go-ethereum/common"
+	"github.com/smartcontractkit/ccip-owner-contracts/pkg/gethwrappers"
+	"github.com/smartcontractkit/ccip-owner-contracts/pkg/proposal/mcms"
+	"github.com/smartcontractkit/ccip-owner-contracts/pkg/proposal/timelock"
+
+	"github.com/smartcontractkit/chainlink/deployment"
+	"github.com/smartcontractkit/chainlink/deployment/common/proposalutils"
+	"github.com/smartcontractkit/chainlink/deployment/common/types"
+	"github.com/smartcontractkit/chainlink/v2/core/gethwrappers/shared/generated/link_token"
+)
+
+var _ deployment.ChangeSet[TransferLinkTokenConfig] = TransferLinkToken
+var _ deployment.ChangeSet[TransferLinkTokenProposalConfig] = TransferLinkTokenProposal
+
+type Transfer struct {
+	To     common.Address
+	Amount *big.Int
+}
+
+type TransferLinkTokenConfig struct {
+	Transfers map[uint64]Transfer
+}
+
+func (c TransferLinkTokenConfig) Validate() error {
+	for k, v := range c.Transfers {
+		if err := deployment.IsValidChainSelector(k); err != nil {
+			return err
+		}
+
+		if v.To == (common.Address{}) {
+			return errors.New("to address must be set")
+		}
+		if v.Amount == nil || v.Amount.Sign() == -1 {
+			return errors.New("amount must be set and positive")
+		}
+	}
+	return nil
+}
+
+// TransferLinkToken transfers link token to the to address on the chain identified by the chainSelector.
```

Comment:
seems more like a QA function not really a configuration change?

Reply:
That's true, we were planning to use this as a test and example for MCMS. Let me check with @RodrigoAD , see what he thinks

Reply:
> seems more like a QA function not really a configuration change?

Configuration is hard to define, in our case, any operation that is required to operate a protocol should be a changeset, as we need all the extra capabilities it provides, e.g. wrap the tx into a MCMS proposal. This operation falls in that category. Besides that, LINK is a simple contract that everyone understands, and will serve as a reference on how to implement changeset operations and proposals